prod-cockpit-rcar.yaml

```diff
@@ -248,6 +248,30 @@ components:
         - "tmp/deploy/images/%{MACHINE}/%{XT_XEN_DTB_NAME}"
         - "tmp/deploy/images/%{MACHINE}/core-image-minimal-%{MACHINE}.ext4"
 
+  cluster:
+    build-dir: "%{YOCTOS_WORK_DIR}"
+    sources:
+      - *COMMON_SOURCES
+    builder:
+      type: yocto
+      work_dir: "%{CLUSTER_BUILD_DIR}"
+      conf:
+        - *COMMON_CONF
+        - *DOMD_DOMU_CONF
+      build_target: cluster-wrapper
+      layers:
+        - "../poky/meta"
+        - "../poky/meta-poky"
+        - "../poky/meta-yocto-bsp"
+        - "../meta-renesas/meta-rcar-gen3"
+        - "../meta-arm/meta-arm-toolchain"
```

Comment:
Nothing provides this layer `meta-arm-toolchain`. I mean there is no `meta-arm` in sources.

Reply:
toolchain is required to build the cluster